<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="Input">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Output">
          <property name="font">
           <font>
            <pointsize>28</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLAbel {
qproperty-alignment: 'AlignVCenter | AlignRight';
border:1px solid gray;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="buttonLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_CE">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>CE</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_C">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>C</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Backspace">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>โซ</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_LeftParen">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>(</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Divide">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>รท</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_7">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>8</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_9">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>9</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_RightParen">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Multiply">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>ร</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>6</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Frac">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Subtract">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_1">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Space">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>_</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Add">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="pushButton_0">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Decimal">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Evaluate">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>15</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
border: 1px solid gray;
}
QPushButton:pressed{
background-color:qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                      stop: 0 #FF7832, stop: 1 #a39a9a)
}</string>
            </property>
            <property name="text">
             <string>=</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup</string>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
